configs/platforms/windows-2012r2-x64.rb

```diff
@@ -9,10 +9,12 @@
   plat.add_build_repository "http://nexus.delivery.puppetlabs.net/service/local/nuget/nuget-pl-build-tools/"
 
   # We don't want to install any packages from the chocolatey repo by accident
+  plat.provision_with "C:/ProgramData/chocolatey/bin/choco.exe update -y chocolatey"
```

Comment:
As we discussed this morning, updating everything to 0.10.7 broke builds, so we should keep this here until we figure out how to fix things.